### PR TITLE
Initial commit for apollo server

### PR DESCRIPTION
**Pull request by CalvinRodo**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+// Type definitions define the "shape" of your data and specify
+// which ways the data can be fetched from the GraphQL server.
+const typeDefs = gql`
+  # Comments in GraphQL are defined with the hash (#) symbol.
+
+  # This "Book" type can be used in other type declarations.
+  type Project {
+    name: String
+    coverage: Int
+    lastUpdated: String
```
Comment (s-laugh) [inline index.js:43]: Would this not be some kind of DateTime?

File: index.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+  }
+];
+
+// Type definitions define the "shape" of your data and specify
+// which ways the data can be fetched from the GraphQL server.
+const typeDefs = gql`
+  # Comments in GraphQL are defined with the hash (#) symbol.
+
+  # This "Book" type can be used in other type declarations.
+  type Project {
```
Comment (s-laugh) [inline index.js:40]: Will need to add a `link` property. (Could be added later)

File: package.json [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/esdc-devx/AppHubApi.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
```
Comment (s-laugh) [inline package.json]: ISC license?

File: index.js (hunk 1 above)
Comment (CalvinRodo) [reply index.js:43]: Graphql doesn't currently have a scalar(leaves on the graph) datetime object by default. 

We'll have to bring in a library to do that.  

Opened an issue here #2

File: index.js (hunk 2 above)
Comment (CalvinRodo) [reply index.js:40]: Opened an issue #3

File: package.json (hunk 3 above)
Comment (CalvinRodo) [reply package.json]: Good catch I'll push a fix

**Review by s-laugh — APPROVED** [review]

**Pull request merged**